ci/scripts/run-check_ci.sh

```diff
@@ -48,54 +49,56 @@ fi
 # Launch experiment
 echo "Launch experiment with Rocoto."
 rocotorun -v "${ROCOTO_VERBOSE:-0}" -w "${xml}" -d "${db}"
-sleep 30
+sleep 10
 if [[ ! -f "${db}" ]]; then
   echo "FATAL ERROR: Rocoto database file ${db} not found, experiment ${pslot} failed, ABORT!"
   exit 2
 fi
 
 # Experiment launched
 rc=99
+set +e
 while true; do
 
   echo "Run rocotorun."
   rocotorun -v "${ROCOTO_VERBOSE:-0}" -w "${xml}" -d "${db}"
 
   # Wait before running rocotostat
-  sleep 30
+  sleep 10
 
   # Get job statistics
   echo "Gather Rocoto statistics"
-  rocotostat_output=$(rocotostat -w "${xml}" -d "${db}" -s | grep -v CYCLE) || true
-  num_cycles=$(echo "${rocotostat_output}" | wc -l) || true
-  num_done=$(echo "${rocotostat_output}" | grep -c Done) || true
-  num_succeeded=$(rocotostat -w "${xml}" -d "${db}" -a | grep -c SUCCEEDED) || true
-  num_failed=$(rocotostat -w "${xml}" -d "${db}" -a | grep -c -E 'FAIL|DEAD') || true
+  # shellcheck disable=SC2312 # We want to use the exit code of the command
+  eval=$("${HOMEgfs}/ci/scripts/utils/rocotostat.py" -w "${xml}" -d "${db}" --export)
+  error_stat=$?
+  eval "${eval}"
```

Comment:
Please find a way to do this without using `eval`. `eval` should nearly always be avoided.

Possible alternatives (from worst to best):
- Have the python script write a script to be sourced (this might be just as bad though)
- Have the python script write the variables to the environment
- Serialize the dict (maybe as a JSON) to either the output or a separate file and then parse it in bash. If you have one key-pair per line, you could use grep to get the right line and then just split on the separator.